.github/workflows/ci.yml

```
@@ -20,28 +23,11 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['20.04.0', '']
+        nxf_ver: ['21.04.0', '21.03.0-edge']
```

```suggestion
        nxf_ver: ['21.04.1', '21.03.0-edge']
```

Could all go to latest stable 21.04.1 with bugfixes